packages/styles/popover.css

```diff
@@ -0,0 +1,118 @@
+:root {
+  --popover-background-color: var(--gray-10);
+  --popover-border-color: var(--gray-90);
+  --popover-text-color: var(--gray-90);
+  --popover-box-shadow: rgba(0, 0, 0, 0.35) 0px 0px 10px 0px;
+  --popover-font-size: var(--text-size-smaller);
+  --popover-line-height: var(--text-size-normal);
+  --popover-padding: 0 var(--space-small);
+  --popover-alert-max-width: 12rem;
+  --popover-alert-min-height: auto;
```

Comment:
This can be omitted. Elements have an implicit height/min-height of `auto` so I don't think we need to set a variable for that.